src/content/developers/docs/bridges/index.md

```diff
@@ -0,0 +1,114 @@
+3. **Integrating a bridge –** This solution doesn’t require the dApp to send users to the external bridge and DEX interfaces. It allows dApps to improve the user onboarding experience. However, this approach has its limitations:
+   - Assessment and maintenance of bridges are hard and time-consuming.
+   - Selecting one bridge creates a single point of failure and dependency.
+   - The dApp is limited by the bridge’s capabilities.
+   - Bridges alone might not be enough. dApps might need DEXs to offer more functionality such as cross-chain swaps.
+4. **Integrating multiple bridges –** This solution solves many problems associated with integrating a single bridge. However, it also has limitations, as integrating multiple bridges is resource-consuming and creates technical and communication overheads for developers – the scarcest resource in crypto.
+5. **Integrating a bridge aggregator –** Another option for dApps is integrating a bridge aggregation solution that gives them access to multiple bridges. Bridge aggregators inherit the strengths of all the bridges and thus are not limited by any single bridge’s capabilities. Notably, the bridge aggregators typically maintain the bridge integrations, which saves the dApp from the hassle of staying on top of the technical and operational aspects of a bridge integration.
+
+   That being said, bridge aggregators also have their limitations. For instance, while they can offer more bridge options, many more bridges are typically available in the market other than those offered on the aggregator's platform. Moreover, just like bridges, bridge aggregators are also exposed to smart contracts and technology risks (more smart contracts = more risks).
+
+If a dApp goes down the route of integrating a bridge or an aggregator, there are different options based on how deep the integration is meant to be. For instance, if it’s only a front-end integration to improve the user onboarding experience, a dApp would integrate the widget. However, if the integration is to explore deeper cross-chain strategies like staking, yield farming, etc., the dApp integrates the SDK or API.
+
+### Deploying a dapp on multiple chains {#deploying-a-dapp-on-multiple-chains}
+
+To deploy a dApp on multiple chains, developers can use development platforms like Alchemy, Hardhat, Truffle, etc. Typically, these platforms come with composable plugins that can enable dApps to go cross-chain. For instance, developers can use a deterministic deployment proxy offered by the hardhat-deploy plugin.
```

Comment:
This should either

1. Provide an example of how to deploy a dapp multichain
2. Link out how to deploy a dapp multichain

Currently, it doesn't really explain anything.

Reply:
Explains a little by noting the plugins, but I still agree it would be nice to have at least a link our or an example.